.github/workflows/link_and_spell_check.yml

```diff
@@ -0,0 +1,48 @@
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+    - main
+
+name: "Check spelling, linting and links"
+
+jobs:
+  check-markdown:
+    runs-on: ubuntu-latest
+    name: "Check spelling, linting and links"
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        wget https://github.com/qir-alliance/.github/archive/refs/heads/main.zip -O clone.zip &&
+        unzip clone.zip && mv .github-main out && rm clone.zip &&
+        mv out/workflow-templates/utils/ utils && rm -r out
+      shell: bash
+    - run: |
+        Get-ChildItem (Join-Path "utils" "problem-matchers") `
+          | ForEach-Object {
+            Write-Host "Adding matcher $_...";
+            Write-Host "::add-matcher::$_";
+          }
+      shell: pwsh
+    - name: markdownlint-cli
+      # cspell:words nosborn
+      uses: nosborn/github-action-markdown-cli@v2.0.0
+      with:
+        files: .
+        config_file: 'utils/md_lint_config.yml'
+    - name: Markdown link check
+      # cspell:words gaurav
+      uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        use-verbose-mode: 'yes'
+      if: ${{ success() || failure() }}
+    - name: "Check spelling in changed files"
+      # cspell:words streetsidesoftware
+      uses: streetsidesoftware/cspell-action@v1.2.4
+      with:
+        inline: warning
+        # Only warn on spelling errors, since there's likely a lot of false
+        # positives with terminology in quantum computing.
+        strict: false
+        incremental_files_only: true
+        config: 'utils/spelling_allowlist.json'
```

Comment:
Likewise, I do not see this file checked in.

Reply:
same as above